actions.yaml

```diff
@@ -20,3 +20,7 @@ update-db:
   required:
     - current-version
     - target-version
+string-params:
```

Comment:
`patch-#bugnumber`

Reply:
I changed to have a general action to fix bugs. 
`apply-patch:
  description: |
    Updates de database to apply the changes needed to fix a bug
  params:
    bug-number:
      type: int
      description: "The number of the bug that needs to be fixed"
  required:
    - bug-number`